src/middlewared/middlewared/etc_files/krb5.keytab.py

```diff
@@ -59,3 +59,8 @@ async def render(service, middleware):
         db_keytabfile = base64.b64decode(keytab['file'].encode())
         db_keytabname = keytab['id']
         await write_keytab(db_keytabname, db_keytabfile)
+
+    if osc.IS_LINUX:
+        if os.path.exists(keytabfile):
```

Comment:
The best practice is
```
        with contextlib.suppress(OSError):
            os.unlink(keytabfile)
```
Because theoretically someone else might remove it between `os.path.exists` and `os.remove`

Reply:
Thanks. I'll file that tip away.